src/subcommands/deployments.ts

```diff
@@ -334,6 +376,183 @@ async function deleteDeployment(args: Args): Promise<void> {
   }
 }
 
+async function redeployDeployment(args: Args): Promise<void> {
+  const api = args.token
+    ? API.fromToken(args.token)
+    : API.withTokenProvisioner(TokenProvisioner);
+  let [deploymentId, mProjectId, mBuild, mProject]: [
+    string,
+    string | undefined,
+    Build | null | undefined,
+    Project | null | undefined,
```

Comment:
Do we need to explicitly write type annotation here, even though `resolveDeploymentId` function has return type declared already?

Reply:
We do, as `resolveDeploymentId` returns `[..., Build | undefined, Project | undefined]`, but we need them also to be null, as we are assigning the return value of `api.getProject()` etc further down.